src/commands/client-kill.json

```
@@ -35,6 +35,10 @@
             [
                 "8.0.0",
                 "Replaced `master` `TYPE` with `primary`. `master` still supported for backward compatibility."
+            ],
+            [
+                "8.1.0",
+                "Accepts multiple IDs as an option."
```

The grammar looks a bit weird. I suggest:

```suggestion
                "`ID` option accepts multiple IDs."
```